include/benchmark/benchmark.h

```diff
@@ -1293,6 +1293,14 @@ struct CPUInfo {
   BENCHMARK_DISALLOW_COPY_AND_ASSIGN(CPUInfo);
 };
 
+struct SystemInformation {
```

Comment:
nit: for consistency with CPUInfo, maybe SystemInfo.

Reply:
Can be done, let me know if you want me proceed with this

Reply:
Yes please.

Reply:
Done